File: fabric-backend-core/src/main/java/io/storydoc/fabric/query/app/tabular/TabularResultSet.java
package io.storydoc.fabric.query.app.tabular;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class TabularResultSet {
    private String tableName;
    private List<Row> rows;
    private List<Column> columns;
    private List<Column> pkColumns;
}

File: fabric-frontend/src/app/api/models/tabular-result-set.ts
/* tslint:disable */
/* eslint-disable */
import { Column } from './column';
import { PagingDto } from './paging-dto';
import { Row } from './row';
export interface TabularResultSet {
  columns?: Array<Column>;
  pagingInfo?: PagingDto;
  pkColumns?: Array<Column>;
  rows?: Array<Row>;
  tableName?: string;
}
